usi/src/main/java/daq/usi/ovs/OpenVSwitch.java

```diff
@@ -0,0 +1,59 @@
+package daq.usi.ovs;
+
+import daq.usi.ISwitchController;
+import daq.usi.ResponseHandler;
+import grpc.Interface;
+import grpc.LinkStatus;
+import grpc.POEStatus;
+import grpc.POESupport;
+import grpc.Power;
+import grpc.SwitchActionResponse;
+
+public class OpenVSwitch implements ISwitchController {
+  private String fauxIface;
+
+  public OpenVSwitch(String fauxIface) {
```

Comment:
The singular fauxIFace doesn't seem right -- what is the ultimate source of this information?  If the ultimate source of this information is the config file, then it should be more the interface map which might have multiple values and/or be non-faux interfaces (e.g. there can be real explicitly named interfaces).

(Also, generally avoid abbreviations, so would be fauxInterface)